Dockerfile

```diff
@@ -1,8 +0,0 @@
-FROM php:7.2-buster
```

Comment:
This Dockerfile was vestigial; we are coordinating them all in the `resources` folder now.